Bug Fix: Replies ping the incorrect user

**Kernel:** 5.15.11-gentoo-x86_64
**libc:** sys-libs/glibc-2.34-r4
**Go:** dev-lang/go-1.17.5
**Terminal:** st

I didn't use the username/password login, I put my token in discordo.toml

Reproduction:
1. click a message with the mouse that you want to reply to
2. press r
3. The input box will indicate that you are replying to the correct user.
4. Type a message and press enter
5. The message is sent as replying to the wrong user.

This is the error I get for the crash:
```
panic: runtime error: index out of range [-1] [recovered]
	panic: runtime error: index out of range [-1]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/application.go:243 +0x4d
panic({0x7af9e0, 0xc000318888})
	runtime/panic.go:1038 +0x215
main.onMessageInputFieldInputCapture(0xc00023fc30)
	github.com/ayntgl/discordo/ui.go:295 +0x55f
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc000036180, 0xc00023fc70)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/box.go:158 +0x3a
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc00023fcb0, 0x6f5adf)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/flex.go:237 +0xd9
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0000bbcb0, 0x10)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/box.go:161 +0x53
github.com/rivo/tview.(*Flex).InputHandler.func1(0x40e9c7, 0x10)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/flex.go:237 +0xd9
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0000bbd40, 0xc00023fdb8)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/box.go:161 +0x53
github.com/rivo/tview.(*Application).Run(0xc000202000)
	github.com/rivo/tview@v0.0.0-20211029090357-1d641bd56588/application.go:335 +0x886
main.main()
	github.com/ayntgl/discordo/main.go:102 +0xb94
```

--- a/ui.go
+++ b/ui.go
@@ -292,7 +292,7 @@ func onMessageInputFieldInputCapture(e *tcell.EventKey) *tcell.EventKey {
 		}
 
 		if len(messagesTextView.GetHighlights()) != 0 {
-			m := selectedChannel.Messages[selectedMessage]
+			_, m := util.FindMessageByID(selectedChannel.Messages, messagesTextView.GetHighlights()[0])
 			d := &discordgo.MessageSend{
 				Content:         t,
 				Reference:       m.Reference(),

